### DISCUSSION TITLE
Flutter pubspec dependencies from git (Flutterfire)

### DISCUSSION THREAD

**Discussion by D4N4PP** [firebase/flutterfire#11705] [General] [answered]
How can I set my pubspec to use Flutterfire dependencies from a forked repository?

`firebase_storage:
   git:
      url: https://github.com/marcuslindemannrohden/flutterfire
firebase_core:
   git:
      url: https://github.com/marcuslindemannrohden/flutterfire
firebase_auth:
   git:
      url: https://github.com/marcuslindemannrohden/flutterfire
cloud_firestore:
   git:
      url: https://github.com/marcuslindemannrohden/flutterfire`

**Comment by pelumibabs** [comment, marked as answer]
Please update your pubspec.yml with below code

`firebase_storage:
    git:
      url: https://github.com/marcuslindemannrohden/flutterfire.git
      path: packages/firebase_storage/firebase_storage

  firebase_auth:
    git:
      url: https://github.com/marcuslindemannrohden/flutterfire.git
      path: packages/firebase_auth/firebase_auth

  cloud_firestore:
    git:
      url: https://github.com/marcuslindemannrohden/flutterfire.git
      path: packages/cloud_firestore/cloud_firestore`
